web/src/hooks/api/config/storage.ts

```diff
@@ -20,24 +20,23 @@
  * find current contact information at www.suse.com.
  */
 
-import { useSuspenseQuery, useQuery } from "@tanstack/react-query";
-import { QueryHookOptions } from "~/types/queries";
-import { extendedConfigQuery } from "~/hooks/api";
+import { useSuspenseQuery } from "@tanstack/react-query";
+import { extendedConfigQuery } from "~/hooks/api/config";
 import { putConfig, Response } from "~/api";
 import { Config } from "~/api/config";
 
-const resetConfig = (data: Config | null): Config => (!data ? {} : { ...data, storage: null });
+const removeStorageConfig = (data: Config | null): Config =>
+  !data ? {} : { ...data, storage: null };
```

Comment:
why so negative. It is usually easier to read for humans in positive form
```suggestion
  data ? { ...data, storage: null } : {};
```